config/deployment/update-github-status.sh

```diff
@@ -4,6 +4,10 @@ curl -X POST \
      -H "Authorization: token $GITHUB_TOKEN" \
      https://api.github.com/repos/travis-ci/travis-web/statuses/$TRAVIS_PULL_REQUEST_COMMIT
 curl -X POST \
+     --data "{\"state\": \"success\", \"target_url\": \"https://`echo $TRAVIS_PULL_REQUEST_BRANCH`-staging.test-deployments.travis-ci.org\", \"description\": \"Visit a org-staging deployment for this commit\", \"context\": \"deployments/org-staging\"}" \
```

Comment:
I decided to use the same domain for this as the `org-production` deployments to avoid having to get another wildcard SSL certificate.

I see that I haven’t explained the actual point of all this, sorry. 😁

These scripts create test deployments for every in-team `travis-web` PR. So you can visit a live version of the application, backed by the true API. They add links to the “checks” at the bottom of the PR. You asked me on Slack about why your org-staging API changes weren’t getting PR deployments, which is because I only set up PR deployments for org-production and com-production. So this PR adds an org-staging deployment:

![image](https://cloud.githubusercontent.com/assets/43280/18790829/4c183a6a-817e-11e6-8355-8b8cc5758031.png)

The PR deployments live at a host of this form: `branch-name.test-deployments.travis-ci.org` (and `.com`). For org-staging, I decided to put them at `branch-name-staging.test-deployments.travis-ci.org`, because they could then share the same Herkou application, Redis database, and SSL certificate. I didn’t have to change the Heroku application at all, which made this PR trivial to add org-staging support.

The bigger question I’m wondering about is whether this slows PR builds down too much. My superficial examination suggests it _doesn’t_, but that makes little sense to me as it has to rebuild the JS and CSS assets for every deployment. Ideally there would also be a com-staging deployment. So I need to look further into this.

BUT ultimately the point of this PR was so you could merge it into [your own](https://github.com/travis-ci/travis-web/pull/678) and have your own stable org-staging deployment without having to worry about it being trampled by anyone. You can still do that, and if it turns out we don’t want this in `master`, just revert it!

Let me know if this makes any sense haha… I also wrote this up [here](https://github.com/travis-pro/team-teal/wiki/travis-web-pull-request-deployments).